src/web/pages/tasks/icons/__tests__/TaskIconsWithSync.jsx

```diff
@@ -28,7 +28,7 @@ describe('TaskIconWithSync component tests', () => {
       taskStatus: TASK_STATUS.stopped,
       expectedTitle: 'Resume',
       expectedFill: false,
-    },
+    }, */
```

Comment:
Is commenting the above lines intentional ?

Reply:
No, I was fixing a test after updating the branch.